src/sentry/models/group.py

```diff
@@ -271,22 +270,19 @@ def from_event_id(self, project, event_id):
             # a Group.
             raise Group.DoesNotExist()
 
-        return Group.objects.get(id=group_id)
+        return self.get(id=group_id)
 
     def filter_by_event_id(self, project_ids, event_id):
```

Comment:
Does this function get used by webhooks? I'm a little confused what this change has to do with the other things in the PR.

Reply:
I was trying to globally clean up some anti-patterns after finding them in webhooks. This change it trying to avoid using the model inside the manager: `Group.objects.filter(...)` -> `self.filter(...)`.